core/include/io.h

```diff
@@ -75,6 +105,10 @@ static inline void io_mask32(vaddr_t addr, uint32_t val, uint32_t mask)
 	io_write32(addr, (io_read32(addr) & ~mask) | (val & mask));
 }
 
+#define io_mask8_barrier(v, c, m)	({ __wmb(); io_mask8(v, c, m); })
```

Comment:
`io_mask8()` does both read and write, is it enough with a write barrier like this?